src/runtime_src/core/edge/user/aie/aiereg.h

```diff
@@ -0,0 +1,367 @@
+      {"Performance_Counter2_Event_Value", 0x00031088},
+      {"Performance_Counter3_Event_Value", 0x0003108C},
+      {"Core_Control", 0x00032000},
+      {"Core_Status", 0x00032004},
+      {"Enable_Events", 0x00032008},
+      {"Reset_Event", 0x0003200C},
+      {"Debug_Control0", 0x00032010},
+      {"Debug_Control1", 0x00032014},
+      {"Debug_Control2", 0x00032018},
+      {"Debug_Status", 0x0003201C},
+      {"PC_Event0", 0x00032020},
+      {"PC_Event1", 0x00032024},
+      {"PC_Event2", 0x00032028},
+      {"PC_Event3", 0x0003202C},
+      {"Error_Halt_Control", 0x00032030},
+      {"Error_Halt_Event", 0x00032034},
+      {"ECC_Control", 0x00032100},
+      {"ECC_Scrubbing_Event", 0x00032110 },
+      {"ECC_Failing_Address", 0x00032120},
+      {"ECC_Instruction_Word_0", 0x00032130},
+      {"ECC_Instruction_Word_1", 0x00032134 },
+      {"ECC_Instruction_Word_2", 0x00032138 },
+      {"ECC_Instruction_Word_3", 0x0003213C },
+      {"Timer_Control", 0x00034000},
+      {"Event_Generate", 0x00034008 },
+      {"Event_Broadcast0", 0x00034010 },
+      {"Event_Broadcast1", 0x00034014 },
+      {"Event_Broadcast2", 0x00034018 },
+      {"Event_Broadcast3", 0x0003401C },
+      {"Event_Broadcast4", 0x00034020 },
+      {"Event_Broadcast5", 0x00034024 },
+      {"Event_Broadcast6", 0x00034028 },
+      {"Event_Broadcast7", 0x0003402C },
+      {"Event_Broadcast8", 0x00034030 },
+      {"Event_Broadcast9", 0x00034034 },
+      {"Event_Broadcast10", 0x00034038 },
+      {"Event_Broadcast11", 0x0003403C },
+      {"Event_Broadcast12", 0x00034040 },
+      {"Event_Broadcast13", 0x00034044 },
+      {"Event_Broadcast14", 0x00034048 },
+      {"Event_Broadcast15", 0x0003404C},
+      {"Event_Broadcast_Block_South_Set", 0x00034050},
+      {"Event_Broadcast_Block_South_Clr", 0x00034054},
+      {"Event_Broadcast_Block_South_Value", 0x00034058},
+      {"Event_Broadcast_Block_West_Set", 0x00034060},
+      {"Event_Broadcast_Block_West_Clr", 0x00034064 },
+      {"Event_Broadcast_Block_West_Value", 0x00034068},
+      {"Event_Broadcast_Block_North_Set", 0x00034070 },
+      {"Event_Broadcast_Block_North_Clr", 0x00034074 },
+      {"Event_Broadcast_Block_North_Value", 0x00034078},
+      {"Event_Broadcast_Block_East_Set", 0x00034080 },
+      {"Event_Broadcast_Block_East_Clr", 0x00034084 },
+      {"Event_Broadcast_Block_East_Value", 0x00034088},
+      {"Trace_Control0", 0x000340D0},
+      {"Trace_Control1", 0x000340D4},
+      {"Trace_Status", 0x000340D8},
+      {"Trace_Event0", 0x000340E0},
+      {"Trace_Event1", 0x000340E4},
+      {"Timer_Trig_Event_Low_Value", 0x000340F0},
+      {"Timer_Trig_Event_High_Value", 0x000340F4},
+      {"Timer_Low", 0x000340F8},
+      {"Timer_High", 0x000340FC },
+      {"Event_Status0", 0x00034200 },
+      {"Event_Status1", 0x00034204 },
+      {"Event_Status2", 0x00034208 },
+      {"Event_Status3", 0x0003420C },
+      {"Combo_event_inputs", 0x00034400 },
```

Comment:
Shouldn't "Event" and "Inputs" be capitalized?

Reply:
Names are taken from standard documentation, and dont want to change the standard name.
http://cervino-doc/HEAD/tile_links/xregdb_me_tile_doc.html

Reply:
Bummer :^(   
Could you triple check the documentation to see if that is what is truly desired.  If so, then is is even more important to be case insensitive.